run.sh

```diff
@@ -275,6 +276,21 @@ EOF
 
 }
 
+runNFSInVM() {
+  if [ "$VM_NFS_CMD" ]; then
```

Comment:
please support a hook "hooks/onRunNFS.sh",

just like the "hooks/onRunSSHFS.sh"